lib/Interpreter/ValuePrinter.cpp

```diff
@@ -862,25 +847,55 @@ namespace cling {
         strm << printUnpackedClingValue(*value);
       }
       strm << "]";
-    } else
-      strm << "<<<invalid>>> " << printAddress(value, '@');
-
+    } else {
+      strm << valuePrinterInternal::kUndefined
```

Comment:
Again - I prefer to keep the distinction of error and invalid `Value`. I.e. I prefer to keep the original code here.